frontend/express/public/core/events/javascripts/countly.details.views.js

```diff
@@ -339,64 +178,44 @@
             }
         }
     });
-
-
-    var getCompareEventsView = function() {
-        var compareEventsVuex = [{
-            clyModel: countlyCompareEvents
-        }];
-
-        return new countlyVue.views.BackboneWrapper({
-            component: EventsWrapper,
-            vuex: compareEventsVuex,
-            templates: [
-                "/core/events/templates/allEvents.html",
-                "/core/events/templates/compareEvents.html"
-            ]
-        });
-    };
-
     var getAllEventsView = function() {
-        var allEventsVuex = [{
-            clyModel: countlyAllEvents
-        }];
-
+        var tabsVuex = countlyVue.container.tabsVuex(["/analytics/events"]);
         return new countlyVue.views.BackboneWrapper({
-            component: EventsWrapper,
-            vuex: allEventsVuex,
-            templates: [
-                "/core/events/templates/allEvents.html",
-                "/core/events/templates/compareEvents.html"
-            ]
+            component: EventsView,
+            vuex: tabsVuex
         });
     };
+    app.route("/analytics/events", "events", function() {
+        var eventsViewWrapper = getAllEventsView();
+        this.renderWhenReady(eventsViewWrapper);
+    });
+
+    app.route("/analytics/events/*tab", "events-tab", function(tab) {
+        var eventsViewWrapper = getAllEventsView();
 
-    app.route("/analytics/events/key/:eventKey", "all-events", function(eventKey) {
         var params = {
-            eventKey: eventKey && eventKey !== "undefined" ? eventKey : undefined
+            tab: tab,
         };
-        var EventAllView = getAllEventsView();
-        EventAllView.params = params;
-        this.renderWhenReady(EventAllView);
+        eventsViewWrapper.params = params;
+        this.renderWhenReady(eventsViewWrapper);
     });
 
-    app.route("/analytics/events", "all-events", function() {
+    countlyVue.container.registerTab("/analytics/events", {
+        priority: 1,
+        name: "eventsDetail",
+        title: "Event Stats",
+        route: "#/" + countlyCommon.ACTIVE_APP_ID + "/analytics/events",
```

Comment:
btw you can also change this route to -
 "#/" + countlyCommon.ACTIVE_APP_ID + "/analytics/events/eventsDetail",